octobot/updater/binary_updater.py

```diff
@@ -56,21 +56,24 @@ async def get_latest_version(self):
         return self._parse_latest_version(await self._get_latest_release_data())
 
     async def update_impl(self) -> bool:
+        # TODO fix binary updater to use release endpoint (assents can't be downloaded from gh directly)
+        self.logger.error(f"Please manually update your OctoBot executable from this page: "
+                          f"{self._get_latest_release_url(False)}")
+        return False
         new_binary_file = await self._download_binary()
         if new_binary_file is not None:
             self._give_execution_rights(new_binary_file)
             self._move_binaries(commands.get_bot_file(), new_binary_file)
             return True
         return False
 
-    def _get_latest_release_url(self):
-        return f"{commons_constants.GITHUB_API_CONTENT_URL}/repos/" \
-               f"{commons_constants.GITHUB_ORGANISATION}/" \
-               f"{constants.OCTOBOT_BINARY_PROJECT_NAME}/releases/latest"
+    def _get_latest_release_url(self, use_api_url):
+        base = f"{commons_constants.GITHUB_API_CONTENT_URL}/repos" if use_api_url else commons_constants.GITHUB_BASE_URL
```

Comment:
I don't understand why we could need `GITHUB_BASE_URL` instead of `GITHUB_API_CONTENT_URL`?

Reply:
One is a link to json data that are parsed and the other is a link to an html page printed in logs